termcolor/src/lib.rs

```diff
@@ -1252,9 +1252,21 @@ impl ColorSpec {
     }
 
     /// Get whether this is intense or not.
+    ///
+    /// On Unix-like systems, this will output the ANSI escape sequence
+    /// that will print the 256-color version of the color specified.
```

Comment:
I'm not an ANSI expert, but [Wikipedia (see 8-bit section)](https://en.wikipedia.org/wiki/ANSI_escape_code) seems to just classify these as "intense" colors. I don't know where the canonical documentation is for this though.

I think your Windows description looks good though!

Reply:
Hmmm... I looked at the same page and got confused by the fact that the escape sequence was for 256-colors. Sorry about that. It does say that he sequence of numbers we're using (9-15) are for high intensity colors.

I also scoured the web and couldn't find the canonical documentation for it. 😞 

I'll just change it to say: "On Unix-like systems, this will output the ANSI escape sequence that will print a high-intensity version of the color specified." (We could use the word gaudy, which is what those colors look like to me but we probably don't want to.)

I understand that's a very vague description, but I cannot come up something better unfortunately.

Reply:
Haha! Yes, that wording sounds good. :) If another enterprising individual can make these descriptions more precise in the future, then that's great, but these will do for now!

Reply:
The macro is misleadingly named. The 'intense' colours are the standard 8 with the bold/intense parameter set (e.g., `\e[1;35m` for intense magenta), or in the 8-bit set (indexed from `0`) they are `8` through `15`. `write_intense!()` simply uses the more comprehensive foreground/background colour sequence which supports the standard 8, the intense 8, and then 240 more pre-defined colours. If you did `write_intense!("5")` you'd get non-intense magenta, and if you did `write_intense!("251")` you'd get a pre-defined medium grey. I think you should even be able to do `write_intense!("255;0;255")` for 24-bit magenta, since it uses the same leading sequence.

Reply:
Yeah basically my understanding is that on Unix "intense" is just another variant of a predefined color, where as on Windows it's an actual setting that is toggled on the console.